packages/cli/src/cli/cogify/action.batch.ts

```diff
@@ -99,7 +98,7 @@ export class ActionBatchJob extends CommandLineAction {
         isCommit: boolean,
     ): Promise<{ jobName: string; jobId: string; memory: number }> {
         const jobName = ActionBatchJob.id(job, quadKey);
-        const alignmentLevels = job.source.resolution - quadKey.length;
+        const alignmentLevels = job.source.resZoom - quadKey.length;
```

Comment:
This is the math, I was talking about a few days ago being wrong

GDAL uses the blocksize for this, so bigger block size = less alignment levels

Reply:
Do you have the formula we want to use?

Reply:
I do not.

Reply:
When adding a datum an `blockFactor` can also now be given which defaults to 2. The `blockSize` is `blockFactor * tms.tileSize`